packages/twenty-front/src/modules/object-record/record-index/options/hooks/useExportTableData.ts

```diff
@@ -0,0 +1,157 @@
+import { useEffect, useState } from 'react';
+import { json2csv } from 'json-2-csv';
+import { useRecoilValue } from 'recoil';
+
+import { useFindManyRecords } from '@/object-record/hooks/useFindManyRecords';
+import { FieldMetadata } from '@/object-record/record-field/types/FieldMetadata';
+import { useRecordTableStates } from '@/object-record/record-table/hooks/internal/useRecordTableStates';
+import { ColumnDefinition } from '@/object-record/record-table/types/ColumnDefinition';
+
+import { useFindManyParams } from '../../hooks/useLoadRecordIndexTable';
+
+export const sleep = (ms: number) =>
```

Comment:
Should we extract this into a global util file ? With the associated test ?

Reply:
Up to you guys. I generally wait for another consumer before "promoting" it to a public utility.